packages/teams-js/src/internal/communication.ts

```diff
@@ -47,15 +47,18 @@ class CommunicationPrivate {
   public static topMessageQueue: MessageRequest[] = [];
   public static nextMessageId = 0;
   public static callbacks: {
-    [id: number]: Function; // (arg1, arg2, ...etc) => void
+    [id: number | string]: Function; // (arg1, arg2, ...etc) => void
```

Comment:
>string [](http://example.com/codeflow?start=18&length=6)

Can you create a UUID type that can be used to hold UUIDs and be passed around throughout the code?